admin_guide/diagnostics_tool.adoc

```diff
@@ -157,36 +161,133 @@ A failure from this check indicates that a significant amount of space in etcd
-a comma-delimited list of check names in your inventory file. For example:
+The threshold limit for total percent usage can be set with a variable in your
+inventory file, for example `max_thinpool_data_usage_percent=90`.
 
-----
-openshift_disable_check=ovs_version,etcd_volume
-----
+|`kibana`, `curator`, `elasticsearch`, `fluentd`
+|This set of checks verifies that Elasticsearch, Fluentd, and Curator pods have
+been deployed and are in a `running` state, and that a connection can be
+established between the control host and the exposed Kibana URL. These checks
+will only run if the `openshift_hosted_logging_deploy` inventory variable is set
+to `true`, to ensure that they are executed in a deployment where a logging
+stack has been deployed.
 
+|`logging_index_time`
+|This check detects higher than normal time delays between log creation and log
+aggregation by Elasticsearch in a logging stack deployment. It fails if a
+user-defined timeout is reached before logs are able to be queried through
+Elasticsearch.
+
+A user-defined timeout may be set by passing the
+`openshift_check_logging_index_timeout_seconds` variable. For example, setting
+`openshift_check_logging_index_timeout_seconds=30` will cause the check to fail
+if a newly-created Kibana log is not able to be queried via Elasticsearch after
+30 seconds.
+
+|`ovs_version`
+|This check ensures that a host has the correct version of Open vSwitch installed
+for the currently deployed version of {product-title}.
+|===
+
+[NOTE]
+====
 A similar set of checks meant to run as part of the installation process can be
 found in
 xref:../install_config/install/advanced_install.adoc#configuring-cluster-pre-install-checks[Configuring Cluster Pre-install Checks]. Another set of checks for checking certificate
 expiration can be found in
 xref:../install_config/redeploying_certificates.adoc#install-config-redeploying-certificates[Redeploying Certificates].
+====
+
+[[admin-guide-health-checks-via-ansible-playbook]]
+=== Running Health Checks via ansible-playbook
+
+To run the *openshift-ansible* health checks using the `ansible-playbook`
+command, specify your cluster's inventory file and run the *_health.yml_*
+playbook:
+
+----
+# ansible-playbook -i <inventory_file> \
+    /usr/share/ansible/openshift-ansible/playbooks/common/openshift-checks/health.yml
+----
+
+To disable specific checks, include the variable `openshift_disable_check` with
+a comma-delimited list of check names in your inventory file before running the
+playbook. For example:
+
+----
+openshift_disable_check=ovs_version,etcd_volume
+----
+
+To set variables that accept user-define values, include the `-e` flag with any
+desired variables in `key=value key=value` format. For example:
+
+----
+# ansible-playbook -i <inventory_file> \
+    /usr/share/ansible/openshift-ansible/playbooks/common/openshift-checks/health.yml \
+    -e "openshift_check_logging_index_timeout_seconds=30 etcd_max_image_data_size_bytes=40000000000"
```

Comment:
I tested just to be sure. We can use a single `-e` if the argument is quoted (because of spaces).

`-e "openshift_check_logging_index_timeout_seconds=30 etcd_max_image_data_size_bytes=40000000000"` is also correct.

And so is using single quotes (avoids shell expansion):

`-e 'openshift_check_logging_index_timeout_seconds=30 etcd_max_image_data_size_bytes=40000000000'`